README.md

```diff
@@ -16,7 +16,8 @@ pip install -r requirements.txt
 ```
 export APP_SETTINGS="config.DevelopmentConfig"
 export DATABASE_URL="sqlite:///dev.db"
-export CF_URL="<<Cloud Foundry URL>>"
+export CF_API_URL="<<Cloud Foundry API URL>>"
+export CF_UAA_URL="<<Cloud Foundry UAA URL>>"
 export CF_USERNAME="<<Username>>"
 export CF_PASSWORD="<<Password>>"
 export SECRET_KEY="<<Secret Key>>"
```

Comment:
Agreed. I've been holding back on changing anything other than deployment here, since it sounds like this whole service might go away before long. But if we do wind up maintaining the repo, then this sounds like a useful change.